go.mod

```diff
@@ -221,8 +233,13 @@ replace (
 	// Use special SDK v0.47.x release with support for both ICS and LSM
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.47.10-ics-lsm
 
+	// use memiavl for faster storage
+	github.com/cosmos/cosmos-sdk/store => github.com/crypto-org-chain/cronos/store v0.0.5-0.20240301012710-e9bb006c8d14
```

Comment:
this replacement is not needed, the `github.com/crypto-org-chain/cronos/store` is not a replacement for the cosmos store package.

Reply:
Is it okay to NOT replace and only use cronos/store for the helper function?

https://github.com/cosmos/gaia/pull/2973/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R61

	memiavlstore "github.com/crypto-org-chain/cronos/store"
	baseAppOptions = memiavlstore.SetupMemIAVL(logger, homePath, appOpts, false, false, baseAppOptions)

Reply:
it's not supposed to replace `cosmos-sdk/store` at all, it's a different package ;D